app/src/test/java/com/github/bsideup/liiklus/GroupVersionTest.java

```diff
@@ -19,7 +19,7 @@
 
 public class GroupVersionTest extends AbstractIntegrationTest {
 
-    private static final int PARTITION = 0;
+    private static final int PARTITION = 1;
```

Comment:
why?

Reply:
In gRPC, integer field is always set (0 by default), changed to 1 to make
sure that we're not using the default value